lib/membrane_rtc_engine/endpoints/hls/mobile_layout_maker.ex

```diff
@@ -33,47 +46,87 @@ defmodule Membrane.RTC.Engine.Endpoint.HLS.MobileLayoutMaker do
 
   defp update_layout(%{tracks: tracks, output_caps: output_caps}) do
     tracks
-    |> Enum.filter(fn {_id, track} ->
+    |> Enum.filter(fn {_id, {track, _caps}} ->
       track.type == :video and not track.metadata["mainPresenter"]
     end)
     |> Enum.with_index()
-    |> Enum.flat_map(fn {{_id, track}, index} ->
-      track_layout = get_track_layout(:basic, index, output_caps)
-      blank_layout = get_track_layout(:blank, index, output_caps)
+    |> Enum.flat_map(fn {{_id, {track, caps}}, index} ->
+      {track_trans, track_layout} = get_track_layout(:basic, index, caps, output_caps)
 
       [
-        {Pad.ref(:input, track.id), track_layout},
-        {Pad.ref(:input, {:blank, track.id}), blank_layout}
+        {{Pad.ref(:input, track.id), track_layout}, {Pad.ref(:input, track.id), track_trans}}
```

Comment:
I think that we could move `Pad.ref(:input, track.id)` to a variable.

Reply:
i will move ii to hls endpoint, because it's not related to layout itself.